PKGBUILD

```diff
@@ -12,33 +11,14 @@ url="https://github.com/archlinux/archinstall"
 license=('GPL')
 depends=('python')
 makedepends=('python-setuptools')
+conflicts=('archinstall' 'archinstall-python' 'python-archinstall-git')
```

Comment:
The conflicts here seem strange. Shouldn't it be:
conflicts=('archinstall' 'python-archinstall' 'python-archinstall-git')

Also, since we're removing the split package, we will need a provides()

Reply:
That's a typo.

Like that?

```
provides=('python-archinstall')
conflicts=('archinstall' 'python-archinstall' 'python-archinstall-git')
```

Reply:
Yes, since we are changing the name to archinstall, provides=('python-archinstall') is enough.